test/unit/math/prim/prob/poisson_binomial_ccdf_log_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <stan/math/prim.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+using stan::math::poisson_binomial_lccdf;
+using vec = Eigen::Matrix<double, Eigen::Dynamic, 1>;
+using mat = Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic>;
```

Comment:
Like with the functions, it's preferred to keep all typedefs and using statements within each ```TEST``` block.